style.css

```diff
@@ -1,3 +1,88 @@
+body {
+  text-align: center;
+}
+
 h1 {
   font-size: 30px;
 }
+
+header {
+  font-size: 20px;
+  margin: 1%;
+  border: solid 2px black;
+  padding: 4px 10px;
+}
+
+header ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  padding: 0 6px;
+}
+
+header li:hover {
+  color: rgb(196, 196, 196);
+}
```

Comment:
- [OPTIONAL] Good job adding a hover state property to the menu list. But I think changing the cursor to a pointer also will give good visual feedback that this is a clickable link. :smile: :+1:  `cursor: pointer;`